app/i18n/en/conf.php

```diff
@@ -180,7 +180,7 @@
 		'reading_view' => 'Switch to reading view',
 		'rss_view' => 'Open RSS view in a new tab',
 		'see_on_website' => 'See on original website',
-		'shift_for_all_read' => '+ <code>shift</code> to mark all articles as read',
+		'shift_for_all_read' => '+ [Shift] to mark previous articles as read<br />+ [Alt] to mark all articles as read',
```

Comment:
`<kbd>Shift</kbd>`?

Reply:
`<kbd>Shift</kbd>` was not looking so good, at least with my font rendering, but let me know if you prefer it.
I considered using ⇧ and ⎇ but did not find them self-explanatory enough. We could combine them with the text, though, like [Shift ⇧] and [Alt ⎇], what do you think?

Reply:
It looks like [⇧ Shift] is more common (with the arrow first)

Reply:
The font rendering looks fine to me, but without additional styles it'll just take the default monospace font face from the browser. Additionally it could be styled cutely like on https://developer.mozilla.org/en-US/docs/Web/HTML/Element/kbd

![Screenshot_2020-03-23_21-54-15](https://user-images.githubusercontent.com/202757/77362220-dfeec580-6d50-11ea-9c39-867430d3c819.png)

I don't think I've ever seen ⎇ before, except for maybe something similar on a road sign. I doubt it'd be very helpful here.

Reply:
![image](https://user-images.githubusercontent.com/1008324/77364373-9bfdbf80-6d54-11ea-8975-e5e653534699.png) 🤓
For reference https://fr.wikipedia.org/wiki/Touche_alt

Reply:
On my laptop keeb it's <kbd>shift ⇧</kbd>, btw.